readme.md

```diff
@@ -64,13 +64,13 @@ export class AppModule {}
 Include the Kirby global styles in your app, e.g., in `src/styles.scss`:
 
 ```css
-@import '~@kirbydesign/designsystem/scss/global-styles';
+@import '@kirbydesign/designsystem/scss/global-styles';
```

Comment:
should we change these to @use now that we are at it?

Reply:
Yes, done!

Reply:
@mictro I am not seeing the change, did you push this one yet?

Reply:
Done